tutorial/simulator.md

```diff
@@ -1 +1,56 @@
 # Link a Simulator to FLARE
+To link a simulator/environment to FLARE, the user needs to derive from the base class `Env` in `<flare_root>/flare/framework/env.py` and implement a standard set of five functions:
+
+```python
+@abstractmethod
+def reset(self):
+    """
+    reset the environment and return a list of initial observations
```

Comment:
reset -> Reset, might need to update the corresponding comments in the codebase as well

Reply:
Waiting for Yang's code to be checked in